src/map/status_effect_container.cpp

```diff
@@ -349,6 +349,12 @@ bool CStatusEffectContainer::CanGainStatusEffect(CStatusEffect* PStatusEffect)
                 return false;
             }
             break;
+        case EFFECT_TERROR:
```

Comment:
Fairly certain this was added in a previous PR

Reply:
a previous one of mine? I had to add this here to get the terror immunity to work. If there is a better way let me know

Reply:
Correct, pretty sure I saw this in a recently-merged PR (or one that's still open)

Reply:
Just rechecked, this is the only one i've made core changes in.